docs/content/node-operation/node-setup.mdx

```diff
@@ -245,7 +249,7 @@ docker run --rm \
 	-v /path/to/data:/data:rw \
 	--name flow-go \
 	--network host \
-	gcr.io/flow-container-registry/execution:v0.25.7 \
+	gcr.io/flow-container-registry/execution:<applicable docker tag> \
```

Comment:
How does this work? where do we define this variable?

Reply:
It's [up above](https://github.com/onflow/flow/pull/1199/files#diff-e77b34dd65e576fd4bf54fa135334555ef2202da09ad167f06616c568899ebe0R185)

>     The actual Docker image tag can be found [here](https://developers.flow.com/nodes/node-operation/past-sporks) for appropriate spork.